cluster/addons/dns/kube2sky/kube2sky.go

```diff
@@ -410,6 +482,15 @@ func watchEndpoints(kubeClient *kclient.Client, ks *kube2sky) kcache.Store {
 	return eStore
 }
 
+func randomID(byteCount int) (string, error) {
```

Comment:
I'm adding a util function to do this, if that lands first we should centralize on that.